docker-compose.yml

```diff
@@ -141,10 +141,12 @@ services:
       - backend-db
     <<: *uncritical-constants-env
     environment:
-      <<: *rabbit-mq-user-and-pass
+      <<: [*rabbit-mq-user-and-pass, *mailjet-api-id-and-secret]
       BACKEND_DB_PASSWORD: ${BACKEND_DB_PASSWORD}
       PROXY_PRIMARY_URL: ${PROXY_PRIMARY_URL:-local-dev.dataland.com}
       BACKEND_MIGRATION_BASELINE_VERSION: ${BACKEND_MIGRATION_BASELINE_VERSION:-1}
+      NOTIFICATION_RECEIVERS_DATA_OWNERSHIP_REQUEST: ${NOTIFICATION_RECEIVERS_DATA_OWNERSHIP_REQUEST}
```

Comment:
Why is this necessary?

Reply:
because we are sending emails from the backend now

Reply:
ok